### ISSUE TITLE
about efficientdet_test.py

### ISSUE THREAD

**Issue by wuminjie12** [zylo117/Yet-Another-EfficientDet-Pytorch#538]
I have trained my datasets with pretrained d2. While i found the val loss going up i stopped it.The test result is as below. There are two classes. The handwrite data and the seal. Only the seal can be detected.
![result](https://user-images.githubusercontent.com/73936469/98095222-3fa37680-1ec5-11eb-88d0-043c6c454e09.jpg)
![微信图片_20201104174219](https://user-images.githubusercontent.com/73936469/98095280-51851980-1ec5-11eb-8dc5-6a1a50b7022a.png)

**Comment by zylo117** [comment]
The mAP result doesn't explicitly show the AP of each class, so it's possible that mAP seems fine while the realistic result sucks.
